libs/cdk/src/lib/utils/directives/focusable-item/focusable-item.directive.ts

```diff
@@ -84,7 +84,7 @@ export class FocusableItemDirective implements HasElementRef {
     private _tabbable = true;
 
     /** @hidden */
-    private _timerId: number | null;
+    private _timerId: ReturnType<typeof setTimeout> | null;
```

Comment:
```suggestion
    private _timerId: ReturnType<typeof global.setTimeout> | null;
```
This should fix it. Basically ts confuses node and web environments

Reply:
sounds good